utils/schemavalidation/schemavalidation.go

```diff
@@ -0,0 +1,43 @@
+//go:generate packr2
+
+package schemavalidation
```

Comment:
I'm on board with shortening this package name to `schema` and moving out of utils. External consumers would then call functions like `schema.ValidateBundle`.

Reply:
See also https://golang.org/doc/effective_go.html#package-names

> By convention, packages are given lower case, single-word names